### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by ManhQuanFX11143**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by ManhQuanFX11143 — COMMENTED** [review]
Damnnn

**Review by NTMQuannuaQMTN — COMMENTED** [review]
Nice

File: index.html
Comment (NTMQuannuaQMTN) [inline index.html:1]: Okay

Comment (ManhQuanFX11143) [inline index.html:1]: Hmm

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (ManhQuanFX11143) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again!!! 🧑‍💻 🤖!</h2>
```

**Pull request merged**